pkg/gatewayserver/redis/registry_test.go

```diff
@@ -153,7 +153,7 @@ func TestRegistry(t *testing.T) {
 			"uplink_count",
 			"last_uplink_received_at",
 			"downlink_count",
-		})
+		}, 0)
 		a.So(err, should.BeNil)
 		retrieved, err = registry.Get(ctx, ids)
 		a.So(err, should.BeNil)
```

Comment:
You probably can add a TTL test here (stored disconnected at with TTL, expect the immediate `Get` to work, sleep the TTL, check that `Get` returns not found).